conjure-test/src/test/errors.rs

```diff
@@ -19,11 +19,17 @@ use crate::types::*;
 
 #[test]
 fn error_serialization() {
-    let error = SimpleError::new("hello", 15, false);
+    #[allow(deprecated)]
```

Comment:
What's deprecated?

Reply:
the `TestObject` field `foo` is marked deprecated in the conjure but is also not optional, I'll just use a different test object

Reply:
Ah gotcha. Yeah probably best to use another object.